src/controllers/confirm.company.controller.ts

```diff
@@ -69,3 +87,20 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
 const getIneligiblePage = (apiError: ApiError): string => {
   return INELIGIBLE_PAGES[apiError.data.status];
 };
+
+const getLatestGaz1Date = async (companyNumber: string, token: string): Promise<string> => {
```

Comment:
would this function be better placed in the company.filing.history.service? you could have it return the latest GAZ1 CompanyFilingHistory and then the controller could pick out the date from it. It should make it more re-usable and keep the controller a bit more 'dumb'.

Reply:
Yep. Makes sense to me. I'm happy to action this change. @ICoxCH do you agree?

Reply:
Changed. Function remains to extract and format the date, but other logic moved to the service.